pkg/manila/dbsync.go

```diff
@@ -100,13 +99,11 @@ func DbSyncJob(instance *manilav1.Manila, labels map[string]string, annotations
 							Command: []string{
 								"/bin/bash",
 							},
-							Args:  args,
-							Image: instance.Spec.ManilaAPI.ContainerImage,
-							SecurityContext: &corev1.SecurityContext{
-								RunAsUser: &runAsUser,
-							},
-							Env:          env.MergeEnvs([]corev1.EnvVar{}, envVars),
-							VolumeMounts: dbSyncMounts,
+							Args:            args,
+							Image:           instance.Spec.ManilaAPI.ContainerImage,
+							SecurityContext: dbSyncSecurityContext(),
```

Comment:
@gouthampacha FYI now this run with its own security context, which is different from cronjobs but it's what we did for glance as well [1].

[1] https://github.com/openstack-k8s-operators/glance-operator/pull/609